integration-test/run.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+set -ex
+
+source extra.sh
```

Comment:
I think you might have forgotten to check in this file (or maybe it contains your secrets).

Reply:
Yeah this is the source of magic secrets.